test-case/check-suspend-resume.sh

```diff
@@ -44,6 +44,10 @@ OPT_HAS_ARG['w']=1         OPT_VAL['w']=5
 OPT_NAME['r']='random'   OPT_DESC['r']="Randomly setup wait/sleep time, range is [$random_min-$random_max], this option will overwrite s & w option"
 OPT_HAS_ARG['r']=0         OPT_VAL['r']=0
 
+# processid is set by check-suspend-resume-with-audio.sh for audio test case
+OPT_NAME['p']='processid'     OPT_DESC['p']='Process ID of aplay or arecord'
```

Comment:
You didn't describe the main purpose/feature you're adding:
```suggestion
OPT_NAME['p']='processid'     OPT_DESC['p']='Fail immediately if this process dies'
```

Ideally, this should be generic and work with any process, not just aplay or arecord.